package.json

```diff
@@ -10,15 +10,14 @@
     "url": "https://github.com/ubermadness/Pinano-Discord-Bot.git"
   },
   "dependencies": {
-    "discord.js": "^11.5.1",
-    "humanize-duration": "^3.21.0",
+    "discord.js": "^11.6.4",
+    "humanize-duration": "^3.22.0",
     "moment": "^2.22.2",
-    "mongodb": "^3.4.1",
-    "node-cron": "^2.0.3",
-    "opusscript": "0.0.6"
```

Comment:
Oh, and this seemed unused, so I removed it.

Reply:
Ah, this was just so I could manually cause PB to stream audio. (e.g. on April Fools' Day)